samples/bluetooth/nrf_dm/README.rst

```diff
@@ -61,7 +61,7 @@ This message contains the following fields:
    * Identifier indicating that the device supports distance measurement.
    * The access address used for measurement.
 
-This scanner/advertiser interaction is used as a synchronisation point.
+This scanner/advertiser interaction is used as a synchronization point.
```

Comment:
btw I noticed that both versions are used in the repository.

Reply:
You are right. Can we go with current version?

Reply:
Needs to be with z